docs/ci.md

```diff
@@ -66,32 +63,38 @@ Here is an example of using the GitHub Web UI to launch an E2E workflow:
 
 The project currently supports the usage of the following runners for the E2E jobs:
 
-* [GitHub built-in GPU runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#specifications-for-gpu-larger-runners) -- referred to as `ubuntu-gpu` in our workflow files Only `e2e.yml` uses this runner.
-* Ephemeral GitHub runners launched on demand on AWS. Most workflows work this way, granting us access to a wider variety of infrastructure at lower cost.
+* Ephemeral GitHub runners launched on demand on AWS. All workflows currently work this way, granting us access to a wider variety of infrastructure at lower cost.
```

Comment:
Is there a reason we need to make this statement here?  Particularly about cost.

Reply:
This is holdover from what already existed - honestly, I'd be fine to ax this whole section on runners if we want, as this information is in other parts of the doc as well.

Reply:
To be clear, when I say "this section" I meant the entire `Current GPU-enabled Runners` section

Reply:
Is it helpful for contributors?  Or stated another way, what would they do with this info?

Reply:
I would vote for either removing or stating it runs on AWS but leaving off the history and cost statements.

Reply:
I removed it